src/plots/linePlot.ts

```diff
@@ -22,6 +31,78 @@ export module Plots {
       this.attr("stroke-width", "2px");
     }
 
+    public x(): Plots.AccessorScaleBinding<X, number>;
+    public x(x: number | Accessor<number>): Line<X>;
+    public x(x: X | Accessor<X>, xScale: Scale<X, number>): Line<X>;
+    public x(x?: number | Accessor<number> | X | Accessor<X>, xScale?: Scale<X, number>): any {
+      if (x == null) {
+        return super.x();
+      } else {
+        if (xScale == null) {
```

Comment:
Different signature because line plot is more specific (there is no `Y` type). I can make it the same if you want for consistency